frontend/src/lib/components/chart/chart-page/encoding/chart-encoding/HeatmapEncoding.svelte

```diff
@@ -119,15 +115,13 @@
 		stringValues={parameters.yChannel === SlicesOrModels.MODELS ? stringValues : []}
 	/>
 </EncodingSection>
-<EncodingSection>
```

Comment:
Kind of weird that there is a system dropdown even if one of the axes is "system".

Reply:
True, only relevant for slice-by-slice.